src/kaltura-player.js

```diff
@@ -650,6 +691,20 @@ class KalturaPlayer extends FakeEventTarget {
     }
   }
 
+  _onChangeSourceEnded(): void {
+    KalturaPlayer._logger.debug('_onChangeSourceEnded');
+    this._initAutoPause();
+    this._initTabVisibility();
+    this._initScrollVisibility();
+  }
+
+  _onPlayerReset(): void {
```

Comment:
why do we need this on a reset event from playkit-js? does playkit-js call reset on itself? AFAIK the only one calling reset on playkit-js is kaltura-player

Reply:
Playkit calls reset in configure and KP calls reset in its own reset and also calls configure in the constructor and the configure.
So if I want to handle all places I need to do it in several places.
That way i am sure this is done in all player reset calls and this is full proof for future change in KP which someone might call configure / reset in other places